src/dispensary.js

```diff
@@ -164,9 +171,10 @@ export default class Dispensary {
         return callback(new Error(err));
       }
 
-      if (response && response.statusCode !== 200) {
+      if (response && response.statusCode && response.statusCode !== 200) {
```

Comment:
Should there be some error handling for when response is there but statusCode is not set. As unlikely as that is.